librocksdb_sys/src/lib.rs

```diff
@@ -366,6 +366,23 @@ extern "C" {
                            valLen: *const size_t,
                            err: *mut *mut c_char)
                            -> *mut u8;
+    pub fn crocksdb_multi_get(db: *const DBInstance,
+                              readopts: *const DBReadOptions,
+                              num_keys: size_t,
+                              keys_list: *const *const u8,
+                              keys_list_sizes: *const size_t,
+                              values_list: *mut *mut u8,
+                              values_list_sizes: *const size_t,
+                              errs: *mut *mut c_char);
+    pub fn crocksdb_multi_get_cf(db: *const DBInstance,
+                                 readopts: *const DBReadOptions,
+                                 cf_handle: *mut DBCFHandle,
+                                 num_keys: size_t,
+                                 keys_list: *const *const u8,
+                                 keys_list_sizes: *const size_t,
+                                 values_list: *mut *mut u8,
```

Comment:
I think it should be `*mut *mut u8`. Both the values_list and its content needs to be dropped explicitly.